drivers/serial/pty.c

```diff
@@ -153,14 +153,10 @@ struct pty_devpair_s
  ****************************************************************************/
 
 static int     pty_semtake(FAR struct pty_devpair_s *devpair);
-#ifndef CONFIG_DISABLE_PSEUDOFS_OPERATIONS
```

Comment:
Why CONFIG_DISABLE_PSEUDOFS_OPERATIONS is not checked anymore? Will it work without pseudo fs ops?

Reply:
Yes, pty can be deleted not only through unlink, but also through driver_unregister. The later api always exist regardless CONFIG_DISABLE_PSEUDOFS_OPERATIONS's setting.